app/oess_pull_azure_interfaces.pl

```diff
@@ -73,12 +84,26 @@ sub reconcile_oess_endpoints {
         );
         next if (!defined $azure_connection);
 
+        my $ep = new OESS::Endpoint(db => $db, model => $endpoint);
+        $ep->load_peers();
+        next if(! $cloud_interconnect_id eq $ep->cloud_interconnect_id());
+
+        my $could_account_id = $azure_connection->{name};
+        my $azure_subnet = find_matching_azure_connection($azure_connection, $cloud_interconnect_id);
+        my $endpoint_peer = get_endpoint_peer($ep, $cloud_interconnect_id, $could_account_id);
+
+        next if(!defined $azure_subnet || !defined $endpoint_peer);
+        
+        if(!(increment_ip($endpoint_peer->{local_ip}, -1) eq $azure_subnet)){
```

Comment:
Can be simplified to `if (increment_ip($endpoint_peer->{local_ip}, -1) ne $azure_subnet) {`

Reply:
simplified